src/apps/Permissions/AppPermissions.js

```diff
@@ -16,6 +17,13 @@ import EntityPermissions from './EntityPermissions'
 import AppRoles from './AppRoles'
 
 class AppPermissions extends React.PureComponent {
+  static propTypes = {
+    address: PropTypes.string.isRequired,
+    app: PropTypes.object,
```

Comment:
I get the impression `app` should be required, but maybe we don't have it if it's still loading?

Reply:
I guess so, took me a while to find this again but here it is:

![image](https://user-images.githubusercontent.com/3072458/48336420-85276300-e660-11e8-9491-3e92e28b06f0.png)

Reply:
Ahh, it's when we're still loading the apps that we don't have this information available. I'll add a comment.